packages/teams-js/src/public/search.ts

```diff
@@ -127,4 +128,14 @@ export namespace search {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.search ? true : false;
   }
+
+  /**
+   * Clear the host M365 application's search box
+   *
+   * @beta
+   */
+  export function closeSearch(): void {
+    ensureInitialized(runtime);
```

Comment:
The other functions in this capability (registerHandlers, unregisterHandlers) both say that they have to be called in the FrameContexts.content context (see those function's ensureInitialized calls). That means that those functions will not work if called outside the context of a full screen app. Why is closeSearch able to be called from any FrameContext, shouldn't it be the same as the other functions?